cl_billing_pricing.module

```diff
@@ -146,7 +216,19 @@ function cl_billing_pricing_field_widget_validate($element, &$form_state) {
     // This thing just uses the default as a single-value rather than array.
     $item['values'] = $element['values']['default']['#value'];
   }
-  $item['expires'] = (!empty($element['expires']['#value']['date']) ? strtotime($element['expires']['#value']['date']) : NULL);
+  
+  if(!empty($element['expires']['#value']['date'])) {
+    // The action expires 86399 seconds after the beginning of expiry day in timezone of the website.
+    $tz_user = date_default_timezone_get();
+    $tz_site = variable_get('date_default_timezone', @date_default_timezone_get());
+    date_default_timezone_set($tz_site);
+    $item['expires'] = strtotime($element['expires']['#value']['date']) + 86399;
```

Comment:
The date input accepts date without time which makes it something like Nov 21, 2016 at 00:00:00. If admin chose that date, I guess they want the license to be valid on that date and expire at Nov 21, 2016 23:59:59. So I am just adding 86399 secs. Do you think I should comment it better?